SonarQube.TeamBuild.PreProcessor/Program.cs

```diff
@@ -49,13 +51,13 @@ private static int Main(string[] args)
         }
 
         /// <summary>
-        ///
+        /// Attempts to work out whether the pre-processor was called by the v0.9 bootstrapper
         /// </summary>
         /// <remarks>The bootstrapper 0.9 downloads the preprocessor to "sqtemp" and the 1.0 to ".sonarqube"</remarks>
         private static bool IsInvokedByBootstrapper0_9()
         {
-            return !String.Equals(
-                FileConstants.RelativePathToTempDir,
+            return String.Equals(
+                V0_9Folder,
```

Comment:
Checking explicitly for "sqtemp" seems safer since that will only happen if the user is using v0.9.